integration_tests/snaps/pip-requirements-file/snapcraft.yaml

```diff
@@ -11,7 +11,9 @@ confinement: strict
 parts:
   python2:
     plugin: python2
+    source: .
```

Comment:
@kyrofa I plan to solve than when I deal with the culprit for https://bugs.launchpad.net/snapcraft/+bug/1606894